internal/e2e/e2eutil/check_files.go

```diff
@@ -0,0 +1,49 @@
+// -----------------------------------------------------------------------------
+// SPDX-FileCopyrightText: Copyright © 2024 bomctl a Series of LF Projects, LLC
+// SPDX-FileName: internal/e2e/e2eutil/check_files.go
+// SPDX-FileType: SOURCE
+// SPDX-License-Identifier: Apache-2.0
+// -----------------------------------------------------------------------------
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// -----------------------------------------------------------------------------
+
+package e2eutil
+
+import (
+	"path"
+
+	"github.com/rogpeppe/go-internal/testscript"
+)
+
+// checkFile is a testscript command that checks the
+// existence of a list of files inside a directory.
+func checkFiles(script *testscript.TestScript, neg bool, args []string) { //nolint: revive
```

Comment:
Does it have to match this signature? Or would this work. I also think I remember reading somewhere that `//nolint` directives shouldn't have a space after the `:`

```suggestion
func checkFiles(script *testscript.TestScript, neg bool, args ...string) { //nolint:revive
```

If it has to match this signature, and it's used in multiple places, can we define a type with it?

```go
type commandFunc func(*testscript.TestScript, bool, []string)
```

Reply:
you are right about the no lint thing, however we have two rules that argue over that, so I've been needing to put two nolint entries to get it to ignore the line